syft/frameworks/torch/hook/hook_args.py

```diff
@@ -174,7 +174,7 @@ def hook_function_args(attr, args, kwargs, return_args_type=False):
         return new_args, kwargs, new_type
 
 
-def build_hook_args_function(args, return_tuple=False):
+def unwrap_args_with_function(args, return_tuple=False):
```

Comment:
Maybe my last review was confusing, I was expecting:
hook_function_args -> unwrap_args_from_function
so for build_hook_args_function we should do:
build_hook_args_function -> build_unwrap_args_from_function